### ISSUE TITLE
The loss value remained around 5

### ISSUE THREAD

**Issue by rongpan123** [fab-jul/L3C-PyTorch#16]
When I trained on CLIC dataset, the loss remained 5. Is it necessary to readjust parameters such as learning rare?

**Comment by fab-jul** [comment]
Hi, well the dataset is quite small, right? So  you should for some initial experiments disable LR decay. Do you have TensorBoard setup to check what’s logged?

**Comment by rongpan123** [comment]
Yes,the dataset is small. I will try to remove the LR decay, thank you.

**Comment by fab-jul** [comment]
Sounds good! I'm curious to see the results. Note that our latest paper compares the L3C from this repo (trained on Open Images) also on CLIC: https://arxiv.org/pdf/2003.10184.pdf

**Comment by fab-jul** [comment]
I'll close this for now but feel free to open if the issue persists.
